app/src/main/java/de/westnordost/streetcomplete/settings/SettingsFragment.java

```diff
@@ -57,6 +96,21 @@ public void onStart()
 		getActivity().setTitle(R.string.action_settings);
 	}
 
+	private Point getShownTile()
+	{
+		SharedPreferences preferences = getActivity().getSharedPreferences(PREF_NAME, Context.MODE_PRIVATE);
+		if(preferences.contains(PREF_LAT) && preferences.contains(PREF_LON))
+		{
+			LngLat pos = new LngLat(
+					Double.longBitsToDouble(preferences.getLong(PREF_LON,0)),
+					Double.longBitsToDouble(preferences.getLong(PREF_LAT,0))
+			);
```

Comment:
You create a LngLat here, and then convert the LngLat to a LatLon. Why not simply create a LatLon in the first place? (The default implementation of that interface is OsmLatLon)